src/utilities/printSchema.ts

```diff
@@ -204,7 +204,12 @@ function printInputObject(type: GraphQLInputObjectType): string {
   const fields = Object.values(type.getFields()).map(
     (f, i) => printDescription(f, '  ', !i) + '  ' + printInputValue(f),
   );
-  return printDescription(type) + `input ${type.name}` + printBlock(fields);
+  return (
+    printDescription(type) +
+    `input ${type.name}` +
+    printOneOf(type.isOneOf) +
```

Comment:
nit: It's overkill to create separate function, ternary operator would be enough here: `type.isOneOf ? ' @oneOf' : ''`.

Reply:
makes sense, was just trying to match the existing pattern for the deprecated directive (which is slightly more complex).  Updated!